### PR TITLE
Added the export to CSV

### PR DESCRIPTION
**Pull request by Yexur**
Added the export to CSV

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -271,6 +271,11 @@
       "integrity": "sha512-jjpyQsKGsOF/wUElNjfPULk+d8PKvJOIXk3IUeBYYmNCy5dMWfrI+JiixYNw8ppKOlcRwWTXFl0B+i5oGrf95Q==",
       "dev": true
     },
+    "@types/papaparse": {
```
Comment (kjrcda) [inline package-lock.json]: should this file have been committed to source control in the first place? I know in root-site we don't

File: src/pages/manage-attendees/manage-attendees.ts [hunk 2]
```diff
@@ -72,7 +74,31 @@ export class ManageAttendeesPage {
...
+            data: csvData
+        });
+
+        var blob = new Blob([csv]);
+        var a = window.document.createElement("a");
+        a.href = window.URL.createObjectURL(blob);
+        a.download = "exportedFitnessClasses.csv";
+        document.body.appendChild(a);
+        a.click();
+        document.body.removeChild(a)
```
Comment (kjrcda) [inline src/pages/manage-attendees/manage-attendees.ts]: magic

File: package-lock.json (hunk 1 above)
Comment (Yexur) [reply package-lock.json]: I shouldn't have committed it

Comment (kjrcda) [reply package-lock.json]: lets delete it then

**Pull request merged**